packages/esm-appointments-app/src/appointments/common-components/appointments-actions.component.tsx

```diff
@@ -22,6 +22,7 @@ interface AppointmentActionsProps {
 
 const AppointmentActions: React.FC<AppointmentActionsProps> = ({ appointment }) => {
   const { t } = useTranslation();
+  const { checkInButton, checkOutButton } = useConfig();
```

Comment:
minor: type the `useConfig` whenever possible: `useConfig<ConfigObject>()`, (and make sure to import the right `ConfigObject` from appointments)

Reply:
I think you mentioned this before, will fix.

Reply:
Handling this in the follow-on PR for O3-2966